swifttool/client.py

```diff
@@ -189,6 +244,14 @@ def generate_script(self, outdir, name='ring_builder.sh', rebalance=True,
         os.chmod(outfile, st.st_mode | stat.S_IEXEC)
         return outfile
 
+    def is_devpresent(self, ringtype, zone, node, port, disk):
+        command = self._ring_search_command(ringtype, zone, node,
+                                            port, disk)
+        with open(os.devnull, 'w') as devnull:
+            rc = subprocess.call(command, shell=True, stdout=devnull,
```

Comment:
Why do you need to devnull the stdout? You don't process it anyway, what's the harm in having the output happen?